storage/google/cloud/storage/_signing.py

```diff
@@ -143,7 +143,11 @@ def generate_signed_url(credentials, resource, expiration,
 
     :type method: str
     :param method: The HTTP verb that will be used when requesting the URL.
-                   Defaults to ``'GET'``.
+                   Defaults to ``'GET'``. If method is ``'RESUMABLE'`` then the
+                   signature will additionally contain the `x-goog-resumable`
+                   header, and the method changed to POST. See the signed URL
+                   docs regarding this flow.
```

Comment:
Can we add an actual link to the signed url documentation?

Reply:
Done.